app/controllers/mixins/image_validation_mixin.rb

```diff
@@ -0,0 +1,34 @@
+module Mixins
+  module ImageValidationMixin
+    extend ActiveSupport::Concern
```

Comment:
I don't think this is needed since this is a normal module without anything extra